apps/voting/contracts/Voting.sol

```diff
@@ -549,30 +551,33 @@ contract Voting is IForwarder, AragonApp {
      * @dev Internal function to cast a vote or object to.
      * @dev It assumes that voter can support or object to the vote
      * @param _voteId The identifier of the vote
-     * @param _supports Whether the voter supports the vote
      * @param _voter The address of the voter
-     * @param _isDelegate Whether the voter is a delegate
+     * @param _voterState The state that voter will apply to vote
+     * @param _votingPower The amount of voting power that voter will apply to vote
      */
-    function _vote(uint256 _voteId, bool _supports, address _voter, bool _isDelegate, uint256 _votingPower) internal {
+    function _vote(uint256 _voteId, address _voter, VoterState _voterState, uint256 _votingPower) internal {
```

Comment:
I prefer to discard this change. Although it removes the extra boolean argument, it requires additional conversions: from `bool` to `VoterState` in the caller method and back from `VoterState` to `bool` within the method itself. This doesn’t simplify the process but rather adds an unnecessary step. While I wasn’t completely satisfied with the previous version, I don’t see any substantial improvements with this new approach